src/test/kotlin/org/stellar/walletsdk/WalletTest.kt

```diff
@@ -29,7 +30,7 @@ internal class WalletTest {
 
       // Public key
       assertEquals(56, publicKey.length)
-      assertTrue(publicKey.startsWith("G"))
+      assertTrue(publicKey.startsWith(""))
```

Comment:
Jw what the thought here is. Is this placeholder until you have a more inclusive test that includes muxed accts, for example?

Reply:
That's my mistake, we should have "G" there. Thanks for catching it!